File: app/src/main/java/com/example/fruitsdiary/exception/NoInternetConnectionException.java
package com.example.fruitsdiary.exception;

import com.example.fruitsdiary.R;

public class NoInternetConnectionException extends FruitDiaryException {

    public NoInternetConnectionException() {
        setErrorMessage(R.string.no_internet_dialog_message);
        setTitle(R.string.no_internet_dialog_title);
    }
}

File: app/src/main/java/com/example/fruitsdiary/exception/NoInternetConnectionException.kt
package com.example.fruitsdiary.exception

import com.example.fruitsdiary.R

class NoInternetConnectionException : FruitDiaryException() {
    init {
        errorMessage = R.string.no_internet_dialog_message
        title = R.string.no_internet_dialog_title
    }
}
